components/usage/pkg/db/workspace.go

```diff
@@ -33,11 +33,11 @@ type Workspace struct {
 	SoftDeletedTime    VarcharTime `gorm:"column:softDeletedTime;type:varchar;size:255;" json:"softDeletedTime"`
 	ContentDeletedTime VarcharTime `gorm:"column:contentDeletedTime;type:varchar;size:255;" json:"contentDeletedTime"`
 
-	Archived  int32 `gorm:"column:archived;type:tinyint;default:0;" json:"archived"`
-	Shareable int32 `gorm:"column:shareable;type:tinyint;default:0;" json:"shareable"`
+	Archived  bool `gorm:"column:archived;type:tinyint;default:0;" json:"archived"`
+	Shareable bool `gorm:"column:shareable;type:tinyint;default:0;" json:"shareable"`
 
 	SoftDeleted sql.NullString `gorm:"column:softDeleted;type:char;size:4;" json:"softDeleted"`
-	Pinned      int32          `gorm:"column:pinned;type:tinyint;default:0;" json:"pinned"`
+	Pinned      bool           `gorm:"column:pinned;type:tinyint;default:0;" json:"pinned"`
 
 	// deleted is reserved for use by db-sync
 	_ int32 `gorm:"column:deleted;type:tinyint;default:0;" json:"deleted"`
```

Comment:
```suggestion
	_ bool `gorm:"column:deleted;type:tinyint;default:0;" json:"deleted"`
```

Reply:
Also,
- `Workspace.ID` and `Workspace.OwnerID` could be of type `uuid.UUID`
- `Workspace.Type` could be an enum [[1](https://github.com/gitpod-io/gitpod/blob/69a093567c834fb76d8ea9d96dc00f2a0d2cff23/components/gitpod-protocol/src/protocol.ts#L566)]

Reply:
Good spot. I'd like to leave this part for later PRs once we start using the data from the model and come across this. This will guide us exactly how we want to encapsulated the enum model for this.